dds/src/implementation/rtps/reader.rs

```diff
@@ -315,7 +335,7 @@ impl RtpsReader {
             .iter()
             .map(|cc| {
                 self.instance_handle_builder
-                    .build_instance_handle(cc.kind, cc.data.as_slice())
+                    .build_instance_handle(cc.kind, &cc.data, &cc.inline_qos)

```

Comment:
The whole in-lineqos?

Reply:
Didn't feel like copying this find all over the place :)